kv-vault/src/config.rs

```diff
@@ -68,6 +75,27 @@ impl Config {
                 Some(certs) => certs.split(',').map(|s| s.trim().to_string()).collect(),
                 _ => Vec::new(),
             },
+            token_increment_ttl: env::var("VAULT_TOKEN_INCREMENT_TTL")
+                .ok()
+                .or_else(|| values.get("token_increment_ttl").cloned())
+                .or_else(|| values.get("TOKEN_INCREMENT_TTL").cloned()),
```

Comment:
Is there a reason we prefix the expected env var with `VAULT_` but don't prefix the expected config value names?

Reply:
Just saving 6 keystrokes IMO, the environment variable prefix is just nice to not accidentally get configured using some other service that has a TOKEN_INCREMENT_TTL

Reply:
Got it 👍